convert/locals.go

```diff
@@ -0,0 +1,62 @@
+package convert
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/hcl/v2/hclwrite"
+)
+
+var localsRegex = regexp.MustCompile(`"([^"]*)"[ ]*=[ ]*"([^"]*)"`)
```

Comment:
Super complicated regex.
Just wondering what example of text it matches so I can understand better why we need that.

Reply:
@razbensimon 
It looks for "key" = "value" entries

Generally, locals may look like this (over complicating it on purpose):
```
{"a" = "b",      "c"=  "d" ,
  "g" = "f" }
```

Reply:
can it contain a more complex type than string?

Reply:
we're specifically parsing the terratag locals. Therefore, it shouldn't. But who knows what folks would do with their files...

Reply:
The regex seems wrong to me. Both key and value of `locals` don't have to include quotes. See https://www.terraform.io/language/values/locals

Also a minor point, we should probably allow for tabs as well as spaces instead of using `[ ]*`

Reply:
You're assuming that it's supposed to handle other locals (which is not the case - as explained above) this should only handle the terratag locals, which are in this style.
Happy to add \t as well.

Reply:
Regarding the locals part, I misunderstood, though this had to do with all locals